spec/std/socket_spec.cr

```diff
@@ -169,15 +169,67 @@ describe Socket::IPAddress do
 end
 
 describe Socket::UNIXAddress do
+  {% if flag?(:linux) %}
+    it "can be abstract on Linux" do
+      path = "/abstract.sock"
+      addr = Socket::UNIXAddress.new('@' + path)
+      addr.abstract?.should be_true
+      addr.path.should eq(path)
+    end
+  {% else %}
+    it "raises for abstract UNIX address on non-Linux" do
+      expect_raises(ArgumentError, /not supported/) do
```

Comment:
Don't need a regexp here, using `"not supported"` as argument will match if the string is included in the error message.

Reply:
right, I though using a `String` would make it a _exact match_ check. Stdlib is well done, it's an inclusion check, thanks :smile:

Reply:
there's at least one other place in this file where a `Regex` is used instead of a `String` for a similar case, but I think it's not in this PR's scope to change that, I'm just mentioning it so that the knowledge is shared (no idea if it's helpful at all ^^)